com.unity.ml-agents/CONTRIBUTING.md

```diff
@@ -40,11 +40,11 @@ contains the following:
 
 ## Environments
 
-We are also actively open to adding community contributed environments as
-examples, as long as they are small, simple, demonstrate a unique feature of the
-platform, and provide a unique non-trivial challenge to modern machine learning
-algorithms. Feel free to submit these environments with a PR explaining the
-nature of the environment and task.
+We are currently not accepting environment contributions directly into ML-Agents.
+However, we believe community created enviornments have a lot of value to the
+community. If you have an interesting enviornment and are willing to share,
```

Comment:
nit: drop "and are willing to share" - it's redundant (they're obviously willing to share) and you use "share" twice in the sentence.